src/internal-links/handler.js

```diff
@@ -9,14 +9,28 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
+/*
+ * Copyright 2024 Adobe. All rights reserved.
```

Comment:
probably it is not needed to have the copyrights header twice (it is already declared).

Reply:
good catch, removed